dashboard/db/migrate/20170131165950_add_uniqueness_to_user_proficiency_user_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniquenessToUserProficiencyUserId < ActiveRecord::Migration[5.0]
+  def up
```

Comment:
Right, added. That said, I'll still ask if you have any recommendations on how to make this reversible using `change` rather than `up` and `down`?